tornado_botocore/base.py

```diff
@@ -73,7 +72,7 @@ def _enable_curl_httpclient(cls):
         if not cls._curl_httpclient_enabled:
             AsyncHTTPClient.configure("tornado.curl_httpclient.CurlAsyncHTTPClient")
             cls._curl_httpclient_enabled = True
-
+    @gen.coroutine
```

Comment:
@michaelwhenderson @jonapgartwohat isn't wrapping a callback based method with coroutine redundant? does this interfere with all other code that uses this method and does not expect a coroutine?

Reply:
Yes, I expect this to cause problems.  You can't use tornado coroutines and callbacks together like this.  At least not in the old versions we have been using. Its callbacks or coroutines not both.